frontend/src/style.css

```
@@ -1 +1,261 @@
+
+.fc-timegrid-now-indicator-arrow {
+  border-color: var(--accent-terracotta) !important;
+  border-width: 6px !important;
+  margin-top: -2px;
+}
+
+.scale-in {
+  animation: scaleIn 0.3s ease-out;
+}
+
+@keyframes scaleIn {
```

Stylelint (17.12.0)

[error] Expected keyframe name "scaleIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)